### ISSUE TITLE
Setting up a Project

### ISSUE THREAD

**Issue by mtthw-meyer** [mtthw-meyer/libdaisy-rust#22]
When creating a new libdaisy-rust project you'll need to copy or create your own memory.x and .cargo/config files to tell the compiler how to build the binary image.

**Comment by x37v** [comment]
a template for [cargo-generate](https://github.com/cargo-generate/cargo-generate) could be nice

**Comment by Spstolar** [comment]
I'm working on this. Trying to base it off of https://github.com/rust-embedded/cortex-m-quickstart

Two questions here:

- Does it matter what repository I'm storing it in? We'll just copy it over to a more official one if I get it working, right? I am working out of https://github.com/Spstolar/libdaisy-rust-quickstart right now.
- Do we have an idea for what `main.rs` should do? My current idea is just to fill it with `blinky.rs`.

**Comment by mtthw-meyer** [comment]
Yes, we'll find a place to put it if you get something working. Any of the examples would be a reasonable base for main.rs, blinky is fine.

**Comment by Spstolar** [comment]
OK, got it running: https://github.com/Spstolar/libdaisy-rust-quickstart . The steps for use are in the README, which can be cleaned up more if needed.

Assuming cargo-generate and the other precursor steps are done

1. `cargo generate --git https://github.com/Spstolar/libdaisy-rust-quickstart`
2. `cargo objcopy --release -- -O binary template.bin`
3. Then (if you're using dfu-util): `dfu-util -a 0 -s 0x08000000:leave -D template.bin`
